### PR TITLE
[SAFE-323] feat: update the resource name when update the core form name

### PR DESCRIPTION
**Pull request by adrianmujica153**

### CODE REVIEW

File: schema/mutation/editForm.ts [hunk 1]
```diff
@@ -144,6 +144,9 @@ export default {
         }
         if (args.name) {
             update.name = args.name;
+            await Resource.findByIdAndUpdate(form.resource, {
+                name: args.name
```
Comment (AntoineRelief) [inline schema/mutation/editForm.ts:148]: @adrianmujica153 did that work only if form is the core of the resource ?

Comment (adrianmujica153) [reply schema/mutation/editForm.ts:148]: Yes, because only the forms that are "core" has the property "resource"

Comment (AntoineRelief) [reply schema/mutation/editForm.ts:148]: I think inherited forms also, I'll check

Comment (adrianmujica153) [reply schema/mutation/editForm.ts]: you are right

**Pull request merged**